### DISCUSSION TITLE
Argocd cli login issue

### DISCUSSION THREAD

**Discussion by sourabhyadavgit** [argoproj/argo-cd#6575] [Q&A]
While configuring argocd with Openldap the web gui works fine but cli login fails.
Web gui works fine and lists out the users info and the groups it is part of but cli throws the error of invalid credentials.

I've also tried using -grcp-web and --grcp-web-root-path option but it's still gives 503 error while using it.

Is cli only allows local users to login via cli?

**Comment by wtam2018** [comment]
@sourabhyadavgit  Argocd SSO login is currently only available in web ui.   See discussion here https://github.com/argoproj/argo-cd/issues/4424  However, if you can get the access token, you can use it in the argocd cli.
